monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -0,0 +1,360 @@
+from shapely.geometry import Point, Polygon
+import shapely.geometry
+from pyproj import Geod, Transformer
+import json
+from pathlib import Path
+from typing import List, NamedTuple
+import arrow
+import datetime
+from datetime import datetime, timedelta
+import uuid
+
+class QueryBoundingBox(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+    name: str
+    shape: Polygon
+    timestamp_before: timedelta
+    timestamp_after: timedelta
+    
+class FlightPoint(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+    lat: float
+    lng: float
+    alt: float
+
+class AircraftPosition(NamedTuple):
+    ''' A object to hold AircraftPosition for Remote ID purposes. For more information see, the definition in the standard: https://github.com/uastech/standards/blob/36e7ea23a010ff91053f82ac4f6a9bfc698503f9/remoteid/canonical.yaml#L1091'''
+
+    lat : float
+    lng : float
+    alt : float
```

Comment:
Can we specify the units and reference frame expected for this field so we can verify whether logic which writes and reads it is correct?

Reply:
Same question as above: What is the reference frame?

Reply:
OK I have cleaned up the comments and referenced in the docstring.